src/main/java/spaceinvaders/group_22/ui/SpaceInvadersUI.java

```diff
@@ -15,17 +16,34 @@
  *
  */
 public class SpaceInvadersUI {
+	/**
+	 * Singleotn instance.
+	 */
+	private static SpaceInvadersUI instance;
```

Comment:
Because I thought in the logger it was thread safe because the logger can have multiple threads. This class is not likely to run into multiple threads I thought. Why does it need to be thread safe?